content/zh/docs/concepts/services-networking/network-policies.md

```diff
@@ -62,33 +62,48 @@ Network policies are implemented by the [network plugin](/docs/concepts/extend-k
+<!--
+By default, a pod is non-isolated for egress; all outbound connections are allowed.  A pod is isolated for egress if there is any NetworkPolicy that both selects the pod and has "Egress" in its `policyTypes`; we say that such a policy applies to the pod for egress.  When a pod is isolated for egress, the only allowed connections from the pod are those allowed by the `egress` list of some NetworkPolicy that applies to the pod for egress.  The effects of those `egress` lists combine additively.
+-->
+
+默认情况下，一个 Pod 的出口是非隔离的，即所有外向连接都是被允许的。 如果有任何的 NetworkPolicy 
+选择该 Pod 并在其 `policyTypes` 中包含 "Egress"，则该 Pod 是出口隔离的，
+我们称这样的策略适用于该 Pod 的出口。当一个 Pod 的出口被隔离时，唯一允许的来自 Pod 的连接是适用
+于出口的 Pod 的某个 NetworkPolicy 的 `egress` 列表所允许的连接。这些 `egress` 列表的效果是相加的。
+
+<!--
+By default, a pod is non-isolated for ingress; all inbound connections are allowed.  A pod is isolated for ingress if there is any NetworkPolicy that both selects the pod and has "Ingress" in its `policyTypes`; we say that such a policy applies to the pod for ingress.  When a pod is isolated for ingress, the only allowed connections into the pod are those from the pod's node and those allowed by the `ingress` list of some NetworkPolicy that applies to the pod for ingress.  The effects of those `ingress` lists combine additively.
 -->
-## 隔离和非隔离的 Pod   {#isolated-and-non-isolated-pods}
 
-默认情况下，Pod 是非隔离的，它们接受任何来源的流量。
+默认情况下，一个 Pod 对入口是非隔离的，即所有入站连接都是被允许的。如果有任何的 NetworkPolicy 
+选择该 Pod 并在其 `policyTypes` 中包含 "Ingress"，则该 Pod 被隔离入口，
+我们称这种策略适用于该 Pod 的入口。 当一个 Pod 的入口被隔离时，唯一允许进入该 Pod
+的连接是来自该 Pod 节点的连接和适用于入口的 Pod 的某个 NetworkPolicy 的 `ingress` 
+列表所允许的连接。这些 `ingress' 列表的效果是相加的。
 
-Pod 在被某 NetworkPolicy 选中时进入被隔离状态。
-一旦名字空间中有 NetworkPolicy 选择了特定的 Pod，该 Pod 会拒绝该 NetworkPolicy
-所不允许的连接。 
-（名字空间下其他未被 NetworkPolicy 所选择的 Pod 会继续接受所有的流量）
+<!--
+Network policies do not conflict; they are additive. If any policy or policies apply to a given pod for a given direction, the connections allowed in that direction from that pod is the union of what the applicable policies allow. Thus, order of evaluation does not affect the policy result.
+
+For a connection from a source pod to a destination pod to be allowed, both the egress policy on the source pod and the ingress policy on the destination pod need to allow the connection. If either side does not allow the connection, it will not happen.
+-->
 
-网络策略不会冲突，它们是累积的。
-如果任何一个或多个策略选择了一个 Pod, 则该 Pod 受限于这些策略的
-入站（Ingress）/出站（Egress）规则的并集。因此评估的顺序并不会影响策略的结果。
+网络策略是相加的，所以不会产生冲突。如果策略适用于 Pod 某一特定方向的流量，Pod 
+在对应方向所允许的连接是适用的网络策略所允许的集合。
+因此，评估的顺序不影响策略的结果。
 
-为了允许两个 Pods 之间的网络数据流，源端 Pod 上的出站（Egress）规则和
-目标端 Pod 上的入站（Ingress）规则都需要允许该流量。
-如果源端的出站（Egress）规则或目标端的入站（Ingress）规则拒绝该流量，
-则流量将被拒绝。
+为了允许从源 Pod 到目的地 Pod 的连接，源 Pod 的出口策略和目的地 Pod 的入口策略都需要允许连接。
```

Comment:
```suggestion
要允许从源 Pod 到目的 Pod 的连接，源 Pod 的出口策略和目的 Pod 的入口策略都需要允许连接。
```

Reply:
Thanks for your carefulness, it help me a lot.